src/graphicswin.cpp

```diff
@@ -74,6 +74,10 @@ const MenuEntry Menu[] = {
 { 1, N_("Select &Edge Chain"),          Command::SELECT_CHAIN,     C|'e',   KN, mEdit  },
 { 1, N_("Select &All"),                 Command::SELECT_ALL,       C|'a',   KN, mEdit  },
 { 1, N_("&Unselect All"),               Command::UNSELECT_ALL,     '\x1b',  KN, mEdit  },
+{ 1,  NULL,                             Command::NONE,             0,       KN, NULL   },
+{ 1, N_("&Line Styles"),                Command::EDIT_LINE_STYLES, 0,       KN, mEdit  },
```

Comment:
All three of these should have a `...` at the end, since they open a new dialog (which in this case gets displayed in the property browser).